cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "projectId": "swcrkh"
```

Comment:
What's the difference in this one and the one in `projects/example-app-cypress/cypress.json`?

Reply:
This file looks to be errant and should be removed.

Reply:
@marienscott this is the file to delete

Reply:
change been made